### PR TITLE
added aboutme.md file

### PR DESCRIPTION
**Pull request by jenieb3**
New about me md file created.

### CODE REVIEW

File: Aboutme.md [hunk 1]
```diff
@@ -0,0 +1,19 @@
+### :open_file_folder: What have I learnt?
+* The different principles of Software Development.
+* How to compile a Java
```
Comment (sohbaker) [inline Aboutme.md]: Should say - how to compile a Java program

File: Aboutme.md [hunk 2]
```diff
@@ -0,0 +1,19 @@
...
+* The different variables in Java
```
Comment (sohbaker) [inline Aboutme.md]: The different type of variables in Java

File: Aboutme.md [hunk 3]
```diff
@@ -0,0 +1,19 @@
...
+
+
+### :confused: What am I confused about?
+* I am not confused about anything, I just need a practical example of SOLID principle of software development. I think once the course progresses I will better understand this. At the moment it's just an abstaract concept.
```
Comment (sohbaker) [inline Aboutme.md]: typo on  the word "abstract". We'll look at creating some code examples of SOLID, but you're right as the course progresses you'll get a better understading of these in practice.

File: Aboutme.md [hunk 4]
```diff
@@ -0,0 +1,19 @@
...
+
+
+
+### Some of the things that I like
+My life is way too exciting to list all the things that I like not in order of priorty.
```
Comment (sohbaker) [inline Aboutme.md]: I think you meant to say something like - "the following are not in order of priority"

**Review by sohbaker — CHANGES_REQUESTED** [review]
This is good! Well done for making good use of different formatting options like headings, bulleted lists and even adding emojis. 

You have a couple of typos so I've added comments.

File: Aboutme.md (hunk 1 above)
Comment (jenieb3) [reply Aboutme.md]: Thank you. I will make the changes

File: Aboutme.md [hunk 5]
```diff
@@ -0,0 +1,19 @@
+### :open_file_folder: What have I learnt?
+* The different principles of Software Development.
+* How to compile a Java programme.
```
Comment (sohbaker) [inline Aboutme.md]: Tends to be written using the Americanised spelling, e.g. 'Java program', 'computer program', etc.

File: Aboutme.md [hunk 6]
```diff
@@ -0,0 +1,19 @@
...
+* The different type of variables in Java.
+
+
+### :confused: What am I confused about?
+* I am not confused about anything, I just need a practical example of SOLID principle of software development. I think once the course progresses I will better understand this. At the moment it's just an abstract concept.
+
+
+
```
Comment (sohbaker) [inline Aboutme.md]: Remove empty lines

File: Aboutme.md [hunk 7]
```diff
@@ -0,0 +1,19 @@
+### :open_file_folder: What have I learnt?
+* The different principles of Software Development.
+* How to compile a Java programme.
+* The different type of variables in Java.
+
+
```
Comment (sohbaker) [inline Aboutme.md]: remove empty line

File: Aboutme.md [hunk 8]
```diff
@@ -0,0 +1,19 @@
...
+### :confused: What am I confused about?
+* I am not confused about anything, I just need a practical example of SOLID principle of software development. I think once the course progresses I will better understand this. At the moment it's just an abstract concept.
+
+
+
+### Some of the things that I like
+My life is way too exciting to list all the things that I like. The following are not in order of priorty:
```
Comment (sohbaker) [inline Aboutme.md]: small typo - 'priority'

**Review by sohbaker — CHANGES_REQUESTED** [review]
Great, thanks for making those changes! A few small things I noticed so I've added to the review. 

As a developer, you would be asked to remove unnecessary empty lines or fix typos in your files, so good practice to be cautious of this now. You shouldn't need to have more than one empty line between paragraphs.

**Review by sohbaker — APPROVED** [review]
Great stuff, well done!

**Comment by jenieb3** [comment]
Thank you.

**Pull request merged**